drivers/tty/serdev/pibridge_trace.h

```diff
@@ -13,7 +13,16 @@
 #define PIBRIDGE_TRACE_MAX_FIFO_DUMP		128
 #define PIBRIDGE_TRACE_MAX_PACKET_DATA_LEN	256
 
-/*  Header */
+/*
+ * pibridge_gate_header_class
+ *
+ * dst: the destination address. Specifies to which RevPi module the packet is
+ *      sent to.
```

Comment:
second "to" is superfluous

Reply:
"RevPi module" is a little imprecise as it sort of suggests the tracepoint (only) applies to I/O modules. In reality the address might as well be a RevPi Core, RevPi Connect or gateway.